src/hotspot/share/gc/g1/g1YoungGCEvacFailureInjector.inline.hpp

```diff
@@ -25,23 +25,24 @@
 #ifndef SHARE_GC_G1_G1YOUNGGCEVACUATIONFAILUREINJECTOR_INLINE_HPP
 #define SHARE_GC_G1_G1YOUNGGCEVACUATIONFAILUREINJECTOR_INLINE_HPP
 
+#include "gc/g1/g1YoungGCEvacFailureInjector.hpp"
+
 #include "gc/g1/g1_globals.hpp"
 #include "gc/g1/g1CollectedHeap.inline.hpp"
-#include "gc/g1/g1YoungGCEvacFailureInjector.hpp"
 
 #ifndef PRODUCT
 
-inline bool G1YoungGCEvacFailureInjector::evacuation_should_fail() {
-  if (!G1EvacuationFailureALot || !_inject_evacuation_failure_for_current_gc) {
+inline bool G1YoungGCEvacFailureInjector::evacuation_should_fail(size_t& counter) {
+  if (!_inject_evacuation_failure_for_current_gc) {
     return false;
   }
   // Injecting evacuation failures is in effect for current GC
   // Access to _evacuation_failure_alot_count is not atomic;
```

Comment:
Comment about non-atomicity seems like it should not be needed anymore.

Reply:
I removed the entire paragraph. There is nothing interesting to tell any more after this change.